src/js/common/browserInfoHelper.ts

```diff
@@ -0,0 +1,174 @@
+    // and then pick the relevant characters based on their position and reformat with a ":"
+    const offset = new Date().toString().split('GMT')[1];
+    const hourOffset = `${offset[0]}${offset[1]}${offset[2]}`;
+    const minuteOffset = `${offset[3]}${offset[4]}`;
+    const formattedUTC = `${hourOffset}:${minuteOffset}`;
+    return formattedUTC;
+};
+
+/**
+ * Function that returns user's timezone offset relative to UTC
+ * @returns {string} UTC concatenated with user's browser's timezone offset
+ */
+export const getTimezone = () => `UTC${getFormattedOffset()}`;
+
+const validateAndGetScreenDetail = (value: number): number | null => {
+    if (isNaN(value)) {
+        return null;
+    } else {
+        return value;
+    }
+};
+
+/**
+ * Function that validates the user's screen's width value, and then returns it.
+ * If it fails validation, it returns null
+ * @returns {number | null} validated value of screen width
+ */
+export const getScreenWidth = () => validateAndGetScreenDetail(screen.width);
+
+/**
+ * Function that validates the user's screen's height value, and then returns it.
+ * If it fails validation, it returns null
+ * @returns {number | null} validated value of screen height
+ */
+export const getScreenHeight = () => validateAndGetScreenDetail(screen.height);
+
+/**
+ * Function that validates the user's device's pixel ratio, and then returns it.
+ * If it fails validation, it returns null
+ * @returns {number | null} validated value of window's devicePixelRatio
+ */
+export const getScreenScalingFactor = () => validateAndGetScreenDetail(window.devicePixelRatio);
+
+/**
+ * Function that validates the user's screen's colorDepth, and then returns it.
+ * If it fails validation, it returns null
+ * @returns {number | null} validated value of screen's colorDepth
+ */
+export const getScreenColourDepth = () => validateAndGetScreenDetail(screen.colorDepth);
+
+/**
+ * Function that validates the user's window's interior width in pixels, and then returns it.
+ * If it fails validation, it returns null
+ * @returns {number | null} validated value of window's innerWidth
+ */
+export const getWindowWidth = () => validateAndGetScreenDetail(window.innerWidth);
+
+/**
+ * Function that validates the user's window's interior height in pixels, and then returns it.
+ * If it fails validation, it returns null
+ * @returns {number | null} validated value of window's innerHeight
+ */
+export const getWindowHeight = () => validateAndGetScreenDetail(window.innerHeight);
+
+/**
+ * The function returns users browser's do not track setting by checking the navigator
+ * and window object for the same
+ * @returns {string} true or false based on users Do Not Track setting
+ */
+export const getBrowserDoNotTrackStatus = () => {
+    const windowVar = window,
+        navigatorVar = navigator;
+    /* eslint-disable eqeqeq */
+    const isBrowserDoNotTrack =
+        (windowVar.doNotTrack && windowVar.doNotTrack == '1') ||
```

Comment:
I know this is just the same as it was before, but is there a reason these aren't `===`? @skodamarthi can you remember?

Reply:
Ya, unfortunately I don't know why. I noticed it as well, but wasn't confident enough to change it to strict equals. My guess is that it is because some browsers implement this as a string and some implement it as a number, but I could be totally wrong there.

Reply:
Apologies for replying late on this. I somehow missed this message before. I don't see a reason why this is not ===, may be a miss on our side. I will change this. Thank you both!